chef/data_bags/crowbar/migrate/crowbar/102_chef-solr.rb

```diff
@@ -12,14 +12,6 @@ def upgrade(ta, td, a, d)
 end
 
 def downgrade(ta, td, a, d)
-  unless ta["chef"].key?("solr_heap")
-    a.delete["chef"]["solr_heap"]
-  end
-  unless ta["chef"].key?("solr_tmpfs")
-    a.delete["chef"]["solr_tmpfs"]
-  end
-  unless ta["chef"]
-    a.delete["chef"]
-  end
+  a.delete("chef") unless ta.key?("chef")
```

Comment:
I'm not sure this is worth the discussion time - but I would feel better if we'd keep it symmetrical. what if some other migration adds chef keys? we'd still have to remove the "solr" keys then, but only those.